src/Lykke.AlgoStore.Service.Security.Core/Utils/FuncExtensions.cs

```diff
@@ -7,7 +7,7 @@ namespace Lykke.AlgoStore.Service.Security.Core.Utils
 {
     public static class FuncExtensions
```

Comment:
Perhaps this should be renamed to LogExtensions to better represent its functionality